web/templates/index.html

```diff
@@ -140,19 +153,19 @@ <h1>botamusique Web Interface</h1>
                 <i class="fas fa-trash-alt" aria-hidden="true"></i> Clear Playlist
             </button>
         </div>
-    </div>
+    </main>
 
     <div class="container mb-3">
         <h2 id="forms">Music Library</h2>
 
         <div class="card mb-3">
             <div class="card-header">
-                <h5 class="card-title">Filters</h5>
+                <h3 class="card-title">Filters</h3>
                 <hr>
                 <div class="row">
                     <div class="col">
-                        <label>Type</label>
-                        <div id="filter-type" class="input-group mb-2">
+                        <fieldset id="filter-type" class="input-group mb-2">
```

Comment:
I think you need to remove `input-group` class from this fieldset to avoid stretching these three buttons.

Reply:
Stretching? Removing that class on my end (using the web inspector in Firefox) seems to have no effect on the buttons :thinking: